clang/test/SemaCXX/cxx98-compat.cpp

```diff
@@ -226,7 +227,8 @@ void TrivialButNonPODThroughEllipsis() {
 }
 
 struct HasExplicitConversion {
-  explicit operator bool(); // expected-warning {{explicit conversion functions are incompatible with C++98}}
+  // FIXME I think we should generate this diagnostic in C++20
```

Comment:
Perhaps this is worth an issue in the issue tracker?  I too think we probably want this firing in >=20 modes as well, and it is a little concerning/confusing that we don't.  Do we do a separate code path/etc for it?  

Not needed to answer here, but again, definitely worth an issue in the tracker.

Reply:
https://github.com/llvm/llvm-project/issues/161689